jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/dialects/SqlDialect.java

```diff
@@ -220,134 +92,162 @@ public String getTableComment() {
     public IdentifierCaseHandling getQuotedIdentifierHandling();
 
     /**
-     * @param identifier The name of an identifier (table or column). If identifiers
-     *                   are case sensitive, the identifier must be passed
-     *                   case-sensitive of course.
+     * @param identifier The name of an identifier (table or column). If identifiers are case sensitive, the identifier
+     *                   must be passed case-sensitive of course.
      * @return the quoted identifier, also if quoting is not required
      */
     public String applyQuote(String identifier);
 
     /**
      * @param identifier The name of an identifier (table or column).
-     * @return the quoted identifier, if this name requires quoting, or the unquoted
-     *         identifier, if no quoting is required.
+     * @return the quoted identifier, if this name requires quoting, or the unquoted identifier, if no quoting is
+     *         required.
      */
     public String applyQuoteIfNeeded(String identifier);
```

Comment:
No dialect makes use of this method, because quotes are always applied. We should remove this method if we don't plan to implement "proper quoting", but just go the easy route and quote always.

Reply:
Removed.